### ISSUE TITLE
Unhandled error event from 3.1.0

### ISSUE THREAD

**Issue by overbyte** [css/csso#335]
We're suddenly getting the following error from csso using 3.1.0 from gulp-csso

using NODE_ENV = env-5.10.1-prebuilt

```
events.js:154
      throw er; // Unhandled 'error' event
      ^
TypeError: Cannot read property 'prev' of undefined
    at Object.module.exports (/mnt/ebs/jenkins/workspace/<project>/node_modules/csso/lib/replace/Number.js:37:45)
    at Object.fn (/mnt/ebs/jenkins/workspace/<project>/node_modules/csso/lib/replace/index.js:19:33)
    at walk (/mnt/ebs/jenkins/workspace/<project>/node_modules/css-tree/lib/walker/index.js:197:20)
    at Object.eval [as MediaFeature] (eval at createWalker (/mnt/ebs/jenkins/workspace/<project>/node_modules/css-tree/lib/walker/index.js:162:25), <anonymous>:4:5)
    at walk (/mnt/ebs/jenkins/workspace/<project>/node_modules/css-tree/lib/walker/index.js:195:39)
    at List.each (/mnt/ebs/jenkins/workspace/<project>/node_modules/css-tree/lib/utils/list.js:143:12)
    at Object.eval [as MediaQuery] (eval at createWalker (/mnt/ebs/jenkins/workspace/<project>/node_modules/css-tree/lib/walker/index.js:162:25), <anonymous>:3:15)
    at walk (/mnt/ebs/jenkins/workspace/<project>/node_modules/css-tree/lib/walker/index.js:195:39)
    at List.each (/mnt/ebs/jenkins/workspace/<project>/node_modules/css-tree/lib/utils/list.js:143:12)
    at Object.eval [as MediaQueryList] (eval at createWalker (/mnt/ebs/jenkins/workspace/<project>/node_modules/css-tree/lib/walker/index.js:162:25), <anonymous>:3:15)
    at walk (/mnt/ebs/jenkins/workspace/<project>/node_modules/css-tree/lib/walker/index.js:195:39)
    at Object.eval [as Atrule] (eval at createWalker (/mnt/ebs/jenkins/workspace/<project>/node_modules/css-tree/lib/walker/index.js:162:25), <anonymous>:6:5)
    at walk (/mnt/ebs/jenkins/workspace/<project>/node_modules/css-tree/lib/walker/index.js:195:39)
    at List.each (/mnt/ebs/jenkins/workspace/<project>/node_modules/css-tree/lib/utils/list.js:143:12)
    at Object.eval [as StyleSheet] (eval at createWalker (/mnt/ebs/jenkins/workspace/<project>/node_modules/css-tree/lib/walker/index.js:162:25), <anonymous>:5:15)
    at walk (/mnt/ebs/jenkins/workspace/<project>/node_modules/css-tree/lib/walker/index.js:195:39)
    at allUp (/mnt/ebs/jenkins/workspace/<project>/node_modules/css-tree/lib/walker/index.js:202:13)
    at module.exports (/mnt/ebs/jenkins/workspace/<project>/node_modules/csso/lib/replace/index.js:17:5)
    at compressChunk (/mnt/ebs/jenkins/workspace/<project>/node_modules/csso/lib/compress.js:69:5)
    at compress (/mnt/ebs/jenkins/workspace/<project>/node_modules/csso/lib/compress.js:143:9)
    at minify (/mnt/ebs/jenkins/workspace/<project>/node_modules/csso/lib/index.js:95:9)
    at Object.minifyStylesheet [as minify] (/mnt/ebs/jenkins/workspace/<project>/node_modules/csso/lib/index.js:124:12)
```

**Comment by dancvogel** [comment]
Me too, but not using gulp-csso

```
An error occurred during minification: 
The execution of the command failed: 
F:\sitesroot\0\Node\node_modules\csso\lib\replace\Number.js:37
    node.value = packNumber(node.value, item.prev);
                                            ^
```

```
TypeError: Cannot read property 'prev' of undefined
    at Object.module.exports (F:\sitesroot\0\Node\node_modules\csso\lib\replace\Number.js:37:45)
    at Object.fn (F:\sitesroot\0\Node\node_modules\csso\lib\replace\index.js:19:33)
    at walk (F:\sitesroot\0\Node\node_modules\css-tree\lib\walker\index.js:197:20)
    at Object.eval [as MediaFeature] (eval at createWalker (F:\sitesroot\0\Node\node_modules\css-tree\lib\walker\index.js:162:25), <anonymous>:4:5)
    at List.walk (F:\sitesroot\0\Node\node_modules\css-tree\lib\walker\index.js:195:39)
    at List.each (F:\sitesroot\0\Node\node_modules\css-tree\lib\utils\list.js:143:12)
    at Object.eval [as MediaQuery] (eval at createWalker (F:\sitesroot\0\Node\node_modules\css-tree\lib\walker\index.js:162:25), <anonymous>:3:15)
    at List.walk (F:\sitesroot\0\Node\node_modules\css-tree\lib\walker\index.js:195:39)
    at List.each (F:\sitesroot\0\Node\node_modules\css-tree\lib\utils\list.js:143:12)
    at Object.eval [as MediaQueryList] (eval at createWalker (F:\sitesroot\0\Node\node_modules\css-tree\lib\walker\index.js:162:25), <anonymous>:3:15)
```

**Comment by lahmatiy** [comment]
Thank you for reporting a bug. Fixed in [3.1.1](https://github.com/css/csso/releases/tag/v3.1.1)

**Comment by overbyte** [comment]
Thanks man - i'll put it back into production
